internships/Outreachy_2023-05/2023-07-20_meeting.md

```diff
@@ -0,0 +1,53 @@
+# Outreachy meeting
+
+### scikit-image project: narrative documentation
+
+- **Time:** Thursday, Jul 20, 2023, 17:00–17:30 UTC
+- **[Join video call via Zoom](https://us06web.zoom.us/j/88060567580?pwd=THRpaWFnSFNwK0Fycy9FVk5RYnV5UT09)**
+- **Notes:** https://hackmd.io/@mkcor/outreachy-meeting
+- **[Archive](https://github.com/scikit-image/skimage-archive/internships/Outreachy_2023-05/)**
+
+**Present:** Ananya, Marianne
+
+**Moderator:** Marianne
+
+
+## Agenda
+
+### Review of the week
+
+- Week 7: [Timeline](https://hackmd.io/@mkcor/outreachy-roadmap)
+    - It's too early to submit a PR!
+    - Getting a hold of the data hasn't been trivial.
+    - Didn't expect this (to this extent). 
+- Got started with `pyklb`.
+    - Reads KLB files into NumPy arrays.
+    - Works at least in a minimal, 'clean' environment.
+    - If needed, will save and load the arrays.
+
+### Career opportunities
+
+- Outreachy's theme for Week 7.
+- Ananya is looking for research internships in academia.
+- Ananya will ask Marianne for recommendation letters when she applies.
+- Open source + open science
+- [DOI](https://www.doi.org/)s aren't just for papers (see [Zenodo](https://zenodo.org/)).
+- See also [ORCiD](https://orcid.org/).
+
+### 2D segmentation
+
+- [Segment human cells](https://scikit-image.org/docs/stable/auto_examples/applications/plot_human_mitosis.html)
+- [Thresholding guide](https://scikit-image.org/docs/stable/auto_examples/applications/plot_thresholding_guide.html)
+- [A tutorial on segmentation](https://emmanuelle.github.io/a-tutorial-on-segmentation.html)?
+
+### Image license
+
+- Origin: https://idr.openmicroscopy.org/webclient/?show=project-502
+- License: CC BY 4.0
+- Ask for relicensing under CC0?
```

Comment:
@stefanv I would dread going this way, since there are 9 authors...

Reply:
Do we need CC0? As far as I am aware, [CC-BY](https://creativecommons.org/licenses/by/4.0/deed.cy) just requires giving credit to the original authors which seems totally do-able and compatible?

Reply:
It would be fine in principle but, as far as I know / to date, *in practice* we cannot guarantee that the image would include the authors metadata all along the process of sharing it, using it in tutorials, ...? Maybe this limitation could be overcome when we have a dedicated "image" type with metadata included?

Reply:
Ah, I didn't think so far as what happens after the image is loaded and just a NumPy array.  If we want to ensure credit in that case to I would argue that it's impossible to assure, even if we add meta data. That can be so easily stripped or lost. Huge point in favor simplifying things with CC0 or another public domain dedication license. :+1:

Reply:
> (...) That can be so easily stripped or lost. Huge point in favor simplifying things with CC0 or another public domain dedication license.

If I recall correctly (and understood correctly back then), that was exactly @stefanv's reasoning and conclusion.

Out of curiosity, which format did you have in mind for just crediting the authors if we were to include their CC-BY image in our data registry?

Reply:
I would have slotted the license and authors into the [README](https://gitlab.com/scikit-image/data/-/blob/master/README.md) and made sure that the authors are credited in every docstring and usage of the dataset. But we can only ensure that in our project.